src/domains/community/detail/components/RelatedPosts.jsx

```
@@ -1,61 +1,40 @@
-
-  useEffect(() => {
-    setPosts([])
-    setHasMore(true)
-  }, [currentId])
+  const { data, fetchNextPage, hasNextPage, isFetchingNextPage, isLoading } = useInfiniteQuery({
+    queryKey: ['relatedPosts', currentId],
+    queryFn: async ({ pageParam = null }) => {
+      const { boards } = await getAllPosts({ last: pageParam, size: 6 })
+      return boards.filter((p) => String(p.boardId) !== String(currentId))
+    },
+    getNextPageParam: (lastPage, allPages) => {
```

ESLint

[error] 'allPages' is defined but never used.

(no-unused-vars)